### ISSUE TITLE
Set brightness on multuple displays at the same time

### ISSUE THREAD

**Issue by Radionz** [blackholeearth/Win10_BrightnessSlider#29]
Thanks for this awsome tool !

I have a request for enhancement, it would be great to have a checkbox or option that allows to set brightness for all monitor with a unique slider.

For example I'm using the tool with a dual screen setup and I would like to set the same brightness on both displays at the same time.

Thank you.

**Comment by pavangayakwad** [comment]
Yes, even I am in need of this very much. please prioritize this feature. Thanks!

**Comment by itsOgden** [comment]
Adding a +1 to the desire for this enhancement. I have 3 monitors and having to shift all of them each time can be tedious. A checkbox to keep them linked would be perfect.

**Comment by Ideekay** [comment]
+1 for this as well. Triple monitors that are the same model and I would love to adjust the brightness for all of them at once.

**Comment by softyoda** [comment]
Same for me (4 monitors)

**Comment by SynergyIncorporated** [comment]
+1 for this as well. Four monitors that are the same model and I would love to adjust the brightness for all of them at once.

**Comment by blackholeearth** [comment]
@softyoda  try v1.7.11 jump to click is added

**Comment by rateforx** [comment]
+1

**Comment by HaroldTien** [comment]
+

**Comment by rateforx** [comment]
This function and more can be found here:
https://github.com/emoacht/Monitorian

**Comment by HMwancheng** [comment]
+1
